### ISSUE TITLE
it doesn't show any extention after following the steps

### ISSUE THREAD

**Issue by LouisMed31** [RatinFX/VegasProFlow#9]
as said in the tital! did all the steps and still it doesn't show any extention added, this would help me  alote in  my work i would appreciate ur help! ty

**Comment by staticanimal** [comment]
same problem here. installed 1.4.4 in all of the suggested folders for manual install, no dice. using vegas pro 18 build 527

**Comment by LouisMed31** [comment]
> 

sadge where is the owner or admin so they can atleast help

**Comment by RatinFX** [comment]
it shows up on my vp18 build 527, all I can say is the following
make sure you:
- search for it under `Tools - Extensions` and NOT `Views - Extensions`
- don't have any typos in your folders
  - ( if relevant - change the folder version number to your VEGAS version )
- don't have the .dll files in a subfolder within the chosen path

**Comment by LouisMed31** [comment]
thank you so fucking much <3

**Comment by RatinFX** [comment]
if something fixed it, please share your solution o/

**Comment by staticanimal** [comment]
> it shows up on my vp18 build 527, all I can say is the following:
> 
> make sure you
> 
> * search for it under `Tools - Extensions` and NOT `Views - Extensions`
> * don't have any typos in your folder names
>   
>   * ( if relevant - change the folder version number to your VEGAS version )
> * don't have the .dll files in a subfolder within the chosen path

yeah that fixed it. i guess at some point between 1.0.3 and 1.4.1 the extension just moved menus for whatever reason because it used to be under view > extensions for me

**Comment by soundphaseytp** [comment]
i cant find the folders necessary to well get it or something else happen because when i quick installed it didn't show in tools > extension can anyone find out what happened?

**Comment by RatinFX** [comment]
If none of the manual paths worked (after double checking every folder name and such), it's a lost cause in my book for unknown reasons I cannot possibly check at this time
